pkg/kubernetes/kubernetes.go

```diff
@@ -86,6 +86,7 @@ func GetKubeConfig(ctx context.Context) (*restclient.Config, error) {
 			log.Errorf("InClusterConfig failed %v", err)
 			return nil, err
 		}
+		config.QPS, config.Burst = getClientThroughput(ctx, true)
 	}
 	return config, nil
```

Comment:
You can remove line 102 in `NewClient()`:

```
	config.QPS, config.Burst = getClientThroughput(ctx, true)
```
Since this function calls `GetKubeConfig` anyway..

Reply:
done.